robottelo/common/helpers.py

```diff
@@ -117,6 +123,27 @@ def get_server_cert_rpm_url():
         get_server_pub_url(), 'katello-ca-consumer-latest.noarch.rpm')
 
 
+def get_host_info(hostname=None):
+    """Get remote host's distribution information
+
+    :param str hostname: Hostname or IP address of the remote host. If ``None``
+        the hostname will be get from ``main.server.hostname`` config.
+
```

Comment:
Can you document the return value? Something like:

    :returns: A tuple in the form ``(distro, major, minor)``. ``major`` and ``minor`` are integers.